qcfractal/interface/models/gridoptimization.py

```diff
@@ -179,7 +178,7 @@ class GridOptimizationRecord(RecordBase):
     # Input data
     initial_molecule: ObjectId = Schema(
         ...,
-        description="Id of the initial molecule in the database."
+        description="ID of the initial molecule in the database."
```

Comment:
I purposefully went through and made it `Id` to match `ObjectId`, we may want to consider changing.